test/cmdlineTests/debug_info_in_yul_and_evm_asm_print_location_only/output

```diff
@@ -194,16 +194,14 @@ object "C_6" {
         code {
             {
                 /// @src 0:60:101
-                let _1 := memoryguard(0x80)
-                mstore(64, _1)
                 if iszero(lt(calldatasize(), 4))
                 {
-                    let _2 := 0
-                    if eq(0x26121ff0, shr(224, calldataload(_2)))
+                    let _1 := 0
+                    if eq(0x26121ff0, shr(224, calldataload(_1)))
                     {
-                        if callvalue() { revert(_2, _2) }
-                        if slt(add(calldatasize(), not(3)), _2) { revert(_2, _2) }
-                        return(_1, _2)
+                        if callvalue() { revert(_1, _1) }
+                        if slt(add(calldatasize(), not(3)), _1) { revert(_1, _1) }
+                        return(memoryguard(0x80), _1)
```

Comment:
Haha this is interesting :)

Reply:
That's exactly what I'd expect, though :-). And nice to see that in this sense apparently this works properly in the presence of ``memoryguard``s.

Reply:
We could actually still think about transforming ``return(x,0)`` -> ``return(0,0)``.

Reply:
That is true! We can do the same for the `call` opcodes.